.env

```diff
@@ -1,2 +1,3 @@
+REACT_APP_WEBMAP_DOMAIN=https://map.treetracker.org
```

Comment:
You will need to remove or update the overwrites in `.env.production` and `.env.staging`, both of which still point at `https://beta...`

Reply:
Will both of them also be `https://map.treetracker.org`?